com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightEditor.cs

```diff
@@ -66,6 +73,12 @@ void OnUndoRedo()
 
         public override void OnInspectorGUI()
         {
+            if (m_AdditionalLightDatas == null)
+            {
+                base.OnInspectorGUI();
```

Comment:
I am not sure we want to do that.

If I get it well here we show the base inspector so the built-in one even if we are in HDRP. It can get confusing for users. 
I think we should add a warning message in the same time, like in the form of an helpbox before the Inspector draws. To be confirmed with UX. ( @MaddalenaUnity )
(same for other types)